indigo_app/templates/place/tabbed_layout_tasks.html

```diff
@@ -0,0 +1,14 @@
+{% extends "main.html" %}
```

Comment:
Please move this into `templates/tasks/` rather than `templates/place/`, it's explicitly NOT place-based.